docs/users/molecule_cookbook.ipynb

```diff
@@ -824,6 +823,9 @@
  ],
  "metadata": {
   "celltoolbar": "Tags",
+  "execution": {
+   "timeout": 120
+  },
```

Comment:
1. That's right, it just gets run in the RTD build step.
2. The rate determining step is `client.query_molecules(molecular_formula="C16H20N3O5")` -  so yeah, it just takes  a minute to query QCArchive

Reply:
I assume (1) means that the notebook breaking will show up as build failures in RTD?

Reply:
Unfortunately no - it's a warning rather than an error and the toolkit docs still have legacy warnings that are difficult or impossible to resolve, so we can't fail on warnings like we do for other docs. It's one of the things I want to sort out when i get around to it.